plasma_cash/client/client.py

```diff
@@ -0,0 +1,14 @@
+from plasma_cash.root_chain.deployer import Deployer
+from .child_chain_service import ChildChainService
+
+
+class Client(object):
+
+    def __init__(self,
+                 root_chain=Deployer().get_contract('RootChain/RootChain.sol'),
+                 child_chain=ChildChainService('http://localhost:8546')):
```

Comment:
recommend the `localhost:8546` to be in config though. People might want to change in different env?